### PR TITLE
Bugfix/mfoc rel movement error

### PR DESCRIPTION
**Pull request by chkettu**
fix for #1834, #1842 (possibly #1841)

### CODE REVIEW

File: drivers/focuser/lacerta_mfoc.cpp [hunk 1]
```diff
@@ -445,12 +437,22 @@ IPState lacerta_mfoc::MoveAbsFocuser(uint32_t targetTicks)
 IPState lacerta_mfoc::MoveRelFocuser(FocusDirection dir, uint32_t ticks)
 {
     // Calculation of the demand absolute position
-    uint32_t targetTicks = FocusAbsPosN[0].value + (ticks * (dir == FOCUS_INWARD ? -1 : 1));
+    uint32_t targetTicks = 0;
+    if (dir == FOCUS_INWARD) {
+      targetTicks = FocusAbsPosN[0].value - ticks;
+    } else {
+      targetTicks = FocusAbsPosN[0].value + ticks;
+    }
```
Comment (knro) [inline drivers/focuser/lacerta_mfoc.cpp:445]: How is this any different from the line you removed above?

File: drivers/focuser/lacerta_mfoc.cpp [hunk 2]
```diff
@@ -474,15 +476,18 @@ uint32_t lacerta_mfoc::GetAbsFocuserPosition()
 
     int nbytes_written = 0;
     int nbytes_read = 0;
+    int count = 0;
+
+    tty_write_string(PortFD, MFOC_cmd, &nbytes_written);
+    LOGF_INFO("CMD <%s>", MFOC_cmd);
```
Comment (knro) [inline drivers/focuser/lacerta_mfoc.cpp:482]: LOGF_DEBUG should be used. LOGF_INFO is for informational messages, not debug.

File: drivers/focuser/lacerta_mfoc.cpp (hunk 1 above)
Comment (chkettu) [reply drivers/focuser/lacerta_mfoc.cpp:445]: The ole line worked on my 32b Linux-Notebook, but not on a 64b-Raspberry PI where it results to "-1" targetTicks. I splitted the line for better debugging options - and with this change it gives the right results. Don't know why...

Comment (knro) [reply drivers/focuser/lacerta_mfoc.cpp:444]: I think a safer way is this:

auto targetTicks = std::clamp(FocusAbsPosN[0].value + (ticks * (dir == FOCUS_INWARD ? -1 : 1)), FocusAbsPosN[0].min, FocusAbsPosN[0].max)

File: drivers/focuser/lacerta_mfoc.cpp (hunk 2 above)
Comment (chkettu) [reply drivers/focuser/lacerta_mfoc.cpp]: changed to debug

File: drivers/focuser/lacerta_mfoc.cpp (hunk 1 above)
Comment (chkettu) [reply drivers/focuser/lacerta_mfoc.cpp:444]: Cool - I'll change this today evening.

Comment (chkettu) [reply drivers/focuser/lacerta_mfoc.cpp]: change commited - plz review again

**Pull request merged**